fortinet_fortigate/plugin.spec.yaml

```
@@ -765,3 +765,30 @@ actions:
         description: An object containing the results of the action
         type: object
         required: true
+  remove_address_object_from_group:
+    title: Remove Address Object from Group
+    description: Removes an address object to an address group
+    input:
+      group:
+        title: Group
+        description: Group name
+        type: string
+        required: true
+        example: blocked IP's
```

Has a better ring to it :)

```suggestion
        example: InsightConnect Block List
```